templates/envs/ropsten/env_base.tmpl

```diff
@@ -3,7 +3,7 @@
 # --- Global configuration ---
 NETWORK=ropsten
 TTD=50000000000000000
-RELAY_URL=https://0xb124d80a00b80815397b4e7f1f05377ccc83aeeceb6be87963ba3649f1e6efa32ca870a88845917ec3f26a8e2aa25c77@builder-relay-ropsten.flashbots.net
+RELAY_URL=https://builder-relay-ropsten.flashbots.net
```

Comment:
why remove the pubkey? it's a required part of the relay URI for mev-boost, not sure if you're setting that somewhere else 🤔

Reply:
I completely miss that. Good catch